test/integration_test.go

```diff
@@ -1214,6 +1215,18 @@ func TestAutoRetryApplyAllDependentModuleRetries(t *testing.T) {
 
 }
 
+func TestAwsProviderPatch(t *testing.T) {
+	t.Parallel()
+
+	stderr := new(bytes.Buffer)
+	rootPath := copyEnvironment(t, TEST_FIXTURE_AWS_PROVIDER_PATCH)
+	modulePath := util.JoinPath(rootPath, TEST_FIXTURE_AWS_PROVIDER_PATCH)
+	err := runTerragruntCommand(t, fmt.Sprintf("terragrunt aws-provider-patch --terragrunt-override-attr region=eu-west-1 --terragrunt-working-dir %s", modulePath), os.Stdout, stderr)
+
+	assert.Nil(t, err)
```

Comment:
NIT: Use `assert.NoError`. The difference is that when there is an error, this will render the error object so you can see the error message (as opposed to some weird pointer).

Reply:
Good call! https://github.com/gruntwork-io/terragrunt/pull/1326/commits/0fc46e35e5eb595dfedf5ddbd8c1b944653b22f7